Ifzan/charcount.js

```diff
@@ -0,0 +1,31 @@
+function abc(q)
+{
+    let count = q.length;
+    console.log(count);
+    let m = new Map();
+    for (let i=0; i<count; i++)
```

Comment:
do you really need to set it to 0 for all elements?

Reply:
to be sure it doesn't hold random value